src/Schedule.h

```diff
@@ -67,6 +67,23 @@ enum class TailStrategy {
     Auto
 };
 
+/** Different ways to handle the case when the start/end of the loops of stages
+ * computed with (fused) are not aligned. */
+enum class AlignStrategy {
```

Comment:
Maybe we should name this `LoopAlignStrategy`, because `AlignStrategy` sounds like something we might want to describe a memory alignment option. It also seems like a more accurate description of what it's doing.